xilem/src/view/sized_box.rs

```diff
@@ -209,3 +221,84 @@ struct BorderStyle {
     width: f64,
     color: Color,
 }
+
+#[derive(Debug, Clone, Copy, PartialEq)]
+pub struct Padding {
```

Comment:
All of this should be documented; we have `expect(missing_docs)` as a temporary crutch, but we want to get things fully documented.

Reply:
I've added documentation now, I'd like if you had a look at it to check that it follows the conventions of the project and that I didn't miss anything.